js/apps/admin-ui/package.json

```diff
@@ -57,7 +57,7 @@
       ]
     },
     "build-lib": {
```

Comment:
The libary build is not triggered on CI, perhaps it should be to prevent regressions from slipping in.

Reply:
Possibly.  But that's a decision for another day.